pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -266,35 +266,12 @@ func GetOpenshiftBootstrapClusterRoles() []rbac.ClusterRole {
 			},
 		},
 		{
-			ObjectMeta: metav1.ObjectMeta{
-				Name: AdminRoleName,
-				Annotations: map[string]string{
-					oapi.OpenShiftDescription: "A user that has edit rights within the project and can change the project's membership.",
-				},
-			},
+			// a role for a namespace level admin.  It is `edit` plus the power to grant permissions to other users.
+			ObjectMeta: metav1.ObjectMeta{Name: "system:openshift:aggregate-to-admin", Labels: map[string]string{"rbac.authorization.k8s.io/aggregate-to-admin": "true"}},
```

Comment:
Finally, I've been waiting for this. Especially when I was trying to figure out the rights changes during 1.8 rebase.